js/src/components/SEMrushRelatedKeyphrasesModal.js

```diff
@@ -55,38 +56,80 @@ class SEMrushRelatedKeyphrasesModal extends Component {
 		this.props.onClose();
 	}
 
+	/**
+	 * Opens the popup window.
+	 *
+	 * @param {event} e The click event.
+	 *
+	 * @returns {void}
+	 */
+	onLinkClick( e ) {
+		const url = "https://oauth.semrush.com/oauth2/authorize?" +
+			"ref=1513012826&client_id=yoast&redirect_uri=https%3A%2F%2Foauth.semrush.com%2Foauth2%2Fyoast%2Fsuccess&response_type=code&scope=user.id";
```

Comment:
Not sure I understand why the actual URL is built in this function while we pass a different URL prop to the link, see https://github.com/Yoast/wordpress-seo/pull/15927/files#diff-08426c832a8a709896f03c0611a210cdR127

Reply:
My fault, I forgot to change the URL in both locations. But you're right in the suggestion of getting the attribute value instead of duplicating the hardcoded string.